app/src/main/AndroidManifest.xml

```diff
@@ -5,9 +5,23 @@
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:supportsRtl="true"
         android:theme="@style/AppTheme">
 
+        <activity android:name="lt.vilnius.tvarkau.StartActivity" />
+
+        <!-- This strangely named alias is here for backwards compatibility.
+             Renaming activity with LAUNCHER category means that shortcuts created by users
+             on their phones with previous version of the app will disappear or stop working. -->
+        <activity-alias
+            android:name="apps.factory.emiestas.MainActivity"
```

Comment:
Do we really need it? [This](http://developer.android.com/guide/topics/manifest/activity-alias-element.html) says `The target must be in the same application as the alias and it must be declared before the alias in the manifest`. If we change app's package name, then it means we'll have different application and those previously created links would not work. Isn't this the case?